build/agent/nginx.conf

```diff
@@ -0,0 +1,29 @@
+load_module /usr/lib/nginx/modules/ngx_http_js_module.so;
+
+events {}
+
+pid /etc/nginx/nginx.pid;
+
+error_log /var/log/nginx/error.log debug;
+
+http {
+    include /etc/nginx/conf.d/*.conf;
+    js_import /usr/lib/nginx/modules/njs/httpmatches.js;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for" ';
+
+    access_log /var/log/nginx/access.log  main;
+
+    # stub status API
+    # needed by the agent in order to collect metrics
+    server {
+        listen 127.0.0.1:8082;
```

Comment:
do we need to expose on the localhost? can we use a unix socket here? 
this way, it will not be possible to get access to it -- only via the unix socket. Also, no way it can collide with any listen we will use in the NGINX config. And, if the the admin use `hostNetworking: true`, the port will not be available on the node.

Reply:
Looks like agent does not support unix sockets for the stub_status API.